static/js/components/dashboard/CourseListCard.js

```diff
@@ -47,22 +48,20 @@ export default class CourseListCard extends React.Component {
     checkout:                        (s: string) => void,
   };
 
+  getRegularPrice = (coursePrice?: CoursePrice): Decimal|null => {
+    return coursePrice ? coursePrice.price : null;
+  }
+
   renderFinancialAidPriceMessage(): ?React$Element<*> {
     const {
       program,
       coupon,
-      prices,
+      coursePrice
     } = this.props;
     const finAidStatus = program.financial_aid_user_info.application_status;
 
     if (FA_TERMINAL_STATUSES.includes(finAidStatus)) {
-      let price;
-      for (const courseRun of sortedCourseRuns(program)) {
-        price = prices.get(courseRun.id);
-        if (price) {
-          break;
-        }
-      }
+      let price = this.getRegularPrice(coursePrice);
```

Comment:
Can you replace `this.getRegularPrice(coursePrice)` with `R.pathOr(null, ['price'], coursePrice)` here and below?